pkg/util/compare.go

```diff
@@ -17,10 +17,25 @@ package util
 import (
 	"fmt"
 	"strings"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/wait"
 
 	"gotest.tools/v3/assert/cmp"
 )
 
+var (
+	// DefaultRetry is the default set of rules
+	// to be followed when retrying for conflicts during
+	// resource update
+	DefaultRetry = wait.Backoff{
```

Comment:
I would not put the default to that (unrelated) util package but move it more closely to the API where it is used. I know, it is used in several different clients where this is used, so finding a proper place is tricky. But this package is for sure not the correct place ;-) For example, if we would have an intermediate package "clients" in `pkg/clients` where all our clients to the backend are collected, then this would be the proper package (we don't have that yet, but maybe that would be a good idea ?)

For now, I would not hesitate to introduce a `pkg/config` with a single file `defaults.go` in a package "config". Might sound like overkill at first, but will help future refactoring if we clearly separate the concerns.

(my general rule of thumb: If one does not know where to put a shared constant, a global `util` package is usually a bad idea (and `util` should be the last choice considered anyway in any case :). Utilities should be independent of a concrete domain and generally reusable for a wide range of scenarios)

Reply:
I wasn't sure about the file to put in. I have moved the variable to the suggested location. I put it in util because the DefaultRetry variable is client-go package is in k8s.io/client-go/util/retry/util.go so why is it incorrect in our case?